packages/clangml/clangml.3.6.0.5/url

```diff
@@ -0,0 +1,2 @@
+archive: "https://github.com/Antique-team/clangml/archive/v3.6.0.6.tar.gz"
```

Comment:
This package has version 3.6.0.5 according to opam (directory structure) but here 3.6.0.6 appears to be downloaded. Would you like to change the archive tag in order to avoid confusion about what opam thinks the version is differing from what the project thinks the version is?